app/models/namespace_group_link.rb

```diff
@@ -23,7 +23,6 @@ class NamespaceGroupLink < ApplicationRecord
   scope :not_expired, -> { where('expires_at IS NULL OR expires_at > ?', Time.zone.now) }
   scope :for_namespace_and_ancestors, lambda { |namespace = nil|
```

Comment:
**nitpick (blocking):** Could you please add a test for this scope.

For testing, I recommend that you use a Group with ancestors and use the `shared_with_group_links` association to get all of them and then ensure that they are all present in `for_namespace_and_ancestors`

Reply:
Added scope testing for both namespace_group_link and member models in 609ccdf0cad480f8926e7af4e2159dffe1f67a26